boards/nucleo-wl55jc/board.c

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (C) 2021  Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_nucleo-wl55jc
+ * @{
+ *
+ * @file        board.c
+ * @brief       Board specific implementations for the Nucleo-wl55jc board
+ *
+ *
+ * @author      Akshai M <akshai.m@fu-berlin.de>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+
+#include "board.h"
+#include "periph/gpio.h"
+
+#include "sx126x.h"
+
+void board_init(void)
+{
+    /* initialize the CPU */
+    cpu_init();
```

Comment:
```suggestion
    board_common_nucleo_init();
```

Otherwise the LED macros won't work. This function also takes care of `cpu_init()`

Reply:
ah, I saw the LED's are initialized below. You could still consider this change though, just in case something changes in the common